tests/unit/ui/components/TestConfirmButton.js

```diff
@@ -89,59 +97,60 @@ describe(__filename, () => {
     );
   });
 
-  it('passes props to the cancel button in the confirmation panel', () => {
-    const cancelButtonText = 'neutral cancel button';
-    const cancelButtonType = 'neutral';
-    const root = render({ cancelButtonText, cancelButtonType });
-
-    // Show the confirmation panel.
-    root.find(Button).simulate('click', createFakeEvent());
-
-    expect(root.find('.ConfirmButton-cancel-button')).toHaveProp(
-      'buttonType',
-      cancelButtonType,
-    );
-    expect(root.find('.ConfirmButton-cancel-button').children()).toHaveText(
-      cancelButtonText,
-    );
-  });
```

Comment:
Why did you remove this test case?

Reply:
hmm..good question...let me double check : )

Reply:
I think I thought this was dupe but I see it's slightly different so I will add this back. thanks for catching this!